armada/handlers/tiller.py

```diff
@@ -156,6 +161,8 @@ def _pre_update_actions(self, actions, namespace):
                     continue
                 LOG.error("Unable to execute name: %s type: %s ", name, type)
         except Exception:
+            raise tiller_exceptions.PreUpdateJobDeleteException(name,
+                                                                namespace)
```

Comment:
This is still a WIP, I will rebase after all of the core functionality has been implemented and make adjustments from there.